### PR TITLE
chore(react): update to new lifecycles

### PR DESCRIPTION
**Pull request by kvet**

### CODE REVIEW

File: packages/dx-grid-core/src/utils/table.test.js [hunk 1]
```diff
@@ -38,15 +37,4 @@ describe('table utils', () => {
       expect(getTableTargetColumnIndex(columnGeometries, 1, 260)).toEqual(3);
     });
   });
-
-  describe('#getAnimations', () => {
-    it('should not return animations if columns are the same', () => {
```
Comment (alexey-semikozov) [inline packages/dx-grid-core/src/utils/table.test.js]: Why you delete this test? The getAnimation function has not cover with a test.

Comment (kvet) [reply packages/dx-grid-core/src/utils/table.test.js]: This test is no longer needed. There are no other tests for this function.

File: packages/dx-react-grid-demos/index.html [hunk 2]
```diff
@@ -8,7 +8,7 @@
   <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
 </head>
 <body>
-  <div class="embedded-demo" data-options='{ "router": "hash", "showThemeSelector": true }'>Loading...</div>
+  <div class="embedded-demo" data-options='{ "router": "hash", "showThemeSelector": true, "frame": true }'>Loading...</div>
```
Comment (gsobolev) [inline packages/dx-react-grid-demos/index.html]: Why are you changing the `frame` parameter here?

Comment (kvet) [reply packages/dx-react-grid-demos/index.html]: Mistake

**Pull request merged**